### DISCUSSION TITLE
UI/UX Improvement

### DISCUSSION THREAD

**Discussion by sanketsohaliya** [subquery/subql#941] [Frontier TestNet - Bug Reporting]
There should be a way to search for indexers by name/address in explorer for delegation. https://frontier.subquery.network/staking/indexers

**Comment by mzxyz** [comment]
@lilian79511

**Comment by sanketsohaliya** [comment]
There should also be a search bar and filters to select optimal plans and service agreements
https://frontier.subquery.network/explorer/project/0x00/service-agreements
